Demo/Demo/SwiftUIComponents/CardPaymentViews/CreateOrderCardPaymentView.swift

```diff
@@ -47,7 +45,9 @@ struct CreateOrderCardPaymentView: View {
                             try await cardPaymentViewModel.createOrder(
                                 amount: "10.00",
                                 selectedMerchantIntegration: DemoSettings.merchantIntegration,
-                                intent: selectedIntent.rawValue
+                                intent: selectedIntent.rawValue,
+                                isVaulting: shouldVaultSelected,
```

Comment:
```suggestion
                                isVaulting: isVaultSelected,
```

Reply:
I wanted to distinguish the isVaulting passed to the viewModel vs the selection in the view.